modules/grafana/templates/dashboards/deployment_panels/_links.json.erb

```diff
@@ -1,5 +1,10 @@
 {
-  "content": "<ul style=\"font-size: 20px;\">\n<li><a target=\"_blank\" href=\"https://errbit.<%= @app_domain %>/\">Errbit</a></li>\n</ul>\n",
+  "content": "<ul style=\"font-size: 20px;\">\n
+      <li><a target=\"_blank\" href=\"https://errbit.<%= @app_domain %>/\">Errbit</a></li>\n
+      <li><a target=\"_blank\" href=\"https://alert.<%= @app_domain %>/\">Icinga</a></li>\n
+      <li><a target=\"_blank\" href=\"https://docs.publishing.service.gov.uk/apps/<%= @app_name %>.html\"><%= @app_name %> documentation</a></li>\n
```

Comment:
Do the dev docs use the same app name that the dashboards do? Or are they based on repo name, which doesn't always match?